wrench/src/reftest.rs

```diff
@@ -114,29 +117,39 @@ impl ReftestManifest {
                 continue;
             }
 
-            let mut items = s.split_whitespace();
+            let items: Vec<&str> = s.split_whitespace().collect();
 
-            match items.next() {
-                Some("include") => {
-                    let include = dir.join(items.next().unwrap());
+            match items[0] {
+                "include" => {
+                    let include = dir.join(items[1]);
 
                     reftests.append(&mut ReftestManifest::new(include.as_path()).reftests);
                 }
-                Some(x) => {
-                    let kind = match x {
-                        "==" => ReftestOp::Equal,
-                        "!=" => ReftestOp::NotEqual,
-                        _ => panic!("unexpected match operator"),
-                    };
-                    let test = dir.join(items.next().unwrap());
-                    let reference = dir.join(items.next().unwrap());
+                x => {
+                    let mut max = 0;
+                    let mut count = 0;
+                    // If the first item is "fuzzy(<val>,<count>)" the positions of the operator
+                    // and paths in the array are offset.
+                    // TODO: This is simple but not great because it does not support having spaces
+                    // in the fuzzy syntax, like between the arguments.
+                    let mut offset = 0;
+                    if x.starts_with("fuzzy(") {
```

Comment:
Here it would be a bit cleaner (and avoid mutability) to write:
```
let (max, count, offset) = if item.starts_with("fuzzy(") {
    ...
    (args[0].parse().unwrap(), args[1].parse().unwrap(), 1)
} else {
    (0, 0, 0)
}
```